<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewer</class>
 <widget class="QMainWindow" name="viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diagram</string>
  </property>
  <property name="windowIcon">
   <iconset resource="reso.qrc">
    <normaloff>:/icons/icons/Chart.png</normaloff>:/icons/icons/Chart.png</iconset>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWebView" name="webView" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="url" stdset="0">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPrint"/>
   <addaction name="actionClose"/>
  </widget>
  <action name="actionPrint">
   <property name="text">
    <string>print</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>close</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="reso.qrc"/>
 </resources>
 <connections/>
</ui>
